AQ-449 [WEB] Main backtest page / Create backtest page

jasonmlam opened the pull request:
This PR also includes story AQ-363.

WillOrtiz7 reviewed (commented):
Looks pretty clean. Here is what I noticed that should be updated.

Since we will have backtests with different statuses (running, complete) we need to display that to the user in the table. There has to be a distinction between a running backtest and complete one, could add a column titled "Status".  Also something to keep in mind for the next ticket is that we dont want to let a user open a backtest that has a status of running.

For creating a backtest we dont want to let the user start a backtest that involved days in the future. So we should block off future dates from being selected.

jasonmlam commented:
> 



> Looks pretty clean. Here is what I noticed that should be updated.
> 
> Since we will have backtests with different statuses (running, complete) we need to display that to the user in the table. There has to be a distinction between a running backtest and complete one, could add a column titled "Status". Also something to keep in mind for the next ticket is that we dont want to let a user open a backtest that has a status of running.
> 
> For creating a backtest we dont want to let the user start a backtest that involved days in the future. So we should block off future dates from being selected.

jasonmlam commented:
Nice pointers.

I addressed your comments. Although I will probably implement the user being able to select completed backtests in my next ticket.

@WillOrtiz7
